src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js

```diff
@@ -56,33 +71,24 @@ const InstrRegForm = () => {
   const onSubmit = evt => {
     evt.preventDefault();
 
-    // TODO: should look similar to this
-    // const data = {
-    //   firstName: formValues.name, // TODO
-    //   lastName: formValues.name, // TODO
-    //   email: formValues.email,
-    //   password: '123456', // TODO
-    //   role_id: 1 // TODO
-    // };
-
-    // fetch('/user/register', {
-    //   method: 'POST',
-    //   headers: {
-    //     Accept: 'application/json',
-    //     'Content-Type': 'application/json'
-    //   },
-    //   body: JSON.stringify(data)
-    // })
-    // .then(() => {
-    //   history.push('/instructor-register-success');
-    // })
-    // .catch(e => {
-    //   console.error(e);
-    //   setFormWarning(e);
-    // });
-
-    // TODO: remove this stub
-    history.push('/instructor-register-success');
+    // This form will be updated with location data will be sent for review
+    axiosWithAuth(authState.idToken)
+      .post('/', {
```

Comment:
We don't want to be pushing up code that doesn't work - and this won't work! We'll want to send this instructor application data to the route described in the BE pull request, `instructors/register`. 😀

Reply:
Ok I believe i just updated it properly! Thank you Paul!